bigquery/cloud-client/test/grantAccessToDataset.test.js

```diff
@@ -0,0 +1,70 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {expect} = require('chai');
+const {
+  getDataset,
+  getEntityId,
+  setupBeforeAll,
+  teardownAfterAll,
+} = require('./config');
+const {grantAccessToDataset} = require('../grantAccessToDataset');
+
+describe('grantAccessToDataset', () => {
+  // Set up fixtures before all tests (similar to pytest's module scope).
+  before(async () => {
+    await setupBeforeAll();
+  });
+
+  // Clean up after all tests.
+  after(async () => {
+    await teardownAfterAll();
+  });
+
+  it('should add entity to access entries', async () => {
+    const dataset = await getDataset();
+    const entityId = getEntityId();
+
+    console.log({dataset});
+    console.log({entityId});
+
+    // Act: Grant access to the dataset.
+    const accessEntries = await grantAccessToDataset(
+      dataset.id,
+      entityId,
+      'READER'
+    );
+
+    // Assert: Check if entity was added to access entries.
+    const updatedEntityIds = accessEntries
```

Comment:
issue: check stdout rather than iterating over the returned types. We can trust that the BigQuery service behaves appropriately. We don't have to check that resources are, in fact created -- we're more interested in ensuring that the sample didn't fail.